app/controllers/jams_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class JamsController < ApplicationController
+  # POST /rooms/:room_hash/jams
+  def create
+    room = Room.find_by!(room_hash: params[:room_id])
+    jam = room.jams.build(jam_params)
+    if jam.save
+      flash[:success] = 'Jam successfully created!'
+    else
+      flash[:danger] = 'A file must be specified.'
```

Comment:
`alert` is more commonly used, hence you can do `flash.alert = `

Reply:
yea i went with success and danger because they match up with the bulma classes (https://bulma.io/documentation/elements/notification/#colors) allowing me to do:

https://github.com/tomekr/jamroulette/pull/5/files#diff-9599427925097c3c66f26ac1e0de5cadR16